homeassistant/components/cloud/account_link.py

```diff
@@ -39,6 +39,10 @@ async def async_provide_implementation(
 ) -> list[config_entry_oauth2_flow.AbstractOAuth2Implementation]:
     """Provide an implementation for a domain."""
     services = await _get_services(hass)
+    if services is None:
+        raise config_entry_oauth2_flow.ImplementationUnavailableError(
```

Comment:
The problem with this is that if account linking would ever be down, it would also prevent local implementations from not being able to load.

Reply:
If a config entry is configured to load the local one, it now does:

* local implementation -> return local one
* Ask cloud -> exception

And gets an exception back, instead of the local implementation.

Reply:
Ah nevermind, looked closer, should be fine.